packages/dashboard/src/pages/Labs/constants.ts

```diff
@@ -13,3 +13,54 @@ export const PLUGIN_IDS = {
     POLL: 'com.maskbook.poll',
     PETS: 'com.maskbook.pets',
 }
+
+type TUTORIAL_URLS_OPT = {
+    [key: string]: string
+}
+
+export const TUTORIAL_URLS_EN: TUTORIAL_URLS_OPT = {
```

Comment:
Suppose you need this configuration to update on the fly. Would you please take a look at https://github.com/DimensionDev/Maskbook-Configuration?